bootstrap-notify/resources/cljsjs/boostrap-notify/common/bootstrap-notify.ext.js

```diff
@@ -0,0 +1,2839 @@
+/**********************************************************************
+ * Extern for jQuery
+ * Generated by http://jmmk.github.io/javascript-externs-generator
+ **********************************************************************/
+var jQuery = {
```

Comment:
The extern file must not redefine vars declared in other extern files, like in jQuery extern. This file should only contain properties added by the library, e.g. `jQuery.notify`, `jQuery.notifyDefaults`, `jQuery.notifyClose` etc.